docs/installation/setup-gce.rst

```diff
@@ -11,7 +11,7 @@ You'll need to setup at least two nodes.
 
 #. Create a new cloud server:
 
-   * Visit the `Google Cloud Console <https://console.cloud.google.com/>`_.
+   * Visit the `Google Cloud Console <https://console.cloud.google.com/start>`_.
```

Comment:
Actually I've found that we have a list of exceptions. So long as https://console.cloud.google.com/ works for everyone here, I propose we just add that as an exception. I'm going to push a few commits on this branch, just so I can re-kick the tests.